hail/src/main/scala/is/hail/io/InputBuffers.scala

```diff
@@ -646,16 +652,15 @@ final class ZstdInputBlockBuffer(blockSize: Int, in: InputBlockBuffer) extends I
     } else {
       val compLen = blockLen - 4
       val decompLen = Memory.loadInt(comp, 0)
-      val ret = Zstd.decompressByteArray(buf, 0, decompLen, comp, 4, compLen)
-      if (Zstd.isError(ret))
-        throw new com.github.luben.zstd.ZstdException(ret)
+      val ret = zstd.decompressByteArray(buf, 0, decompLen, comp, 4, compLen)
```

Comment:
If we trust these methods to throw then we don't need to bind their return values.

Reply:
good point! For the record, they can fail, but the `throw` the exception. The return value is always the number of bytes, never an error code.